### ISSUE TITLE
Accessible (reduced motion) animations don't work as intended

### ISSUE THREAD

**Issue by camelwater** [camelwater/whats-wrong-with-emery#1]
Reduced motion framer-motion animations are off.

I think this is due to the variants check: the `useReducedMotion` hook is initially false, so when it first renders the motion component it takes the initial value of the wrong variant (the non-reduced motion variant). This causes the initial placement of the components to be off (x or y offset), since the non-reduced motion variants involve sliding and whatnot.

Basically, I don't think variants work very well with the `useReducedMotion` hook.

I could try using the `initial` and `animate` props directly (it makes no difference, just some extra effort compared to variants). It's just a bit annoying, and I'll have to test this theory to see if it is correct.

**Comment by camelwater** [comment]
The initial variant is indeed the issue.
